core/logger/buf.go

```diff
@@ -49,7 +49,7 @@ func insertSorted(recs []LogRecord, in []LogRecord) []LogRecord {
 			recs = append(recs, r)
 			continue
 		}
-		copy(recs[:i+1], recs[i:])
```

Comment:
lol, this is so very wrong. Glad you found this.

Reply:
i must have been on crack or something when i added it